src/darwin/Framework/CHIPTests/TestHelpers/MTRTestPerControllerStorage.h

```diff
@@ -38,6 +38,9 @@ NS_ASSUME_NONNULL_BEGIN
     removeValueForKey:(NSString *)key
         securityLevel:(MTRStorageSecurityLevel)securityLevel
           sharingType:(MTRStorageSharingType)sharingType;
+
+// For testing - direct access to the current count of keys in storage
+@property (nonatomic, readonly) NSUInteger count;
```

Comment:
could we contain this to a testing-only header?

Reply:
This is already in a testing header

Reply:
that it is...

Reply:
(the `// For testing` part of the comment might be unnecessary in a testing-only header)